Client/ViewModels/MonthlyBudgetViewModel.razor.cs

```diff
@@ -19,27 +25,63 @@ public MonthlyBudgetViewModel(HttpClient apiClient, NavigationManager navManager
             monthlyBudgetService = mbService;
         }
 
-        public async Task GetLatestMonthlyBudgetForUser() {
+        public async Task SetupLatestMonthlyBudgetForUser() {
             
-            appState.monthlyBudgetModel = await monthlyBudgetService.GetLatestMonthlyBudget();
-            /*DateTime today = DateTime.Now;
-            int month = today.Month;
-            int year = today.Year;
-            appState.monthlyBudgetModel = await monthlyBudgetService.GetMonthlyBudget(month, year);
-            if(appState.monthlyBudgetModel == null)
-            {
-                appState.monthlyBudgetModel.Income = 0;
-                appState.monthlyBudgetModel.Outgoing = 0;
-                appState.monthlyBudgetModel.Remaining = 0;
-            }*/
+           await GetAllMonthlyBudgetsForUser();
+
+           GetAvailableYearsForUser();
```

Comment:
Personally, I prefer these "getters" to return the data rather than set them internally. But, that's your choice 😄

Reply:
Changed them all to return a value